src/api/searchApi.ts

```diff
@@ -93,3 +93,15 @@ export const fetchNviCandidate = async (identifier: string) => {
 
   return getNviCandidates.data;
 };
+
+export const searchForDoiInNva = async (results: number, from: number, doi = '') => {
+  const paginationQuery = `results=${results}&from=${from}`;
+  const searchQuery = doi ? `doi=${doi}` : '';
+
+  const fullQuery = [paginationQuery, searchQuery].filter(Boolean).join('&');
```

Comment:
```suggestion
Denne skal vel ta over for hele `fetchResults` etter hvert, så jeg tenker det gir mening å forbrede til det allerede nå, ved at man kan sende inn søketerm for hva som helst (men bare støtte for DOI i første omgang

export const fetchResults2 = async (results: number, from: number, {doi: ''}) => {
  const paginationQuery = `results=${results}&from=${from}`;
  const doiQuery = doi ? `doi=${doi}` : '';

  const fullQuery = [paginationQuery, doiQuery].filter(Boolean).join('&');
```

Reply:
da kan vi utvide `{doi: ''}` tiol `{doi: '', title: ''}` for å støtte søk på tittel også i fremtiden, i stedet for å lage mange ulike funksjoner som alle bare takler en use case

Reply:
Får en error når `doi` sendes inn som et object.

Reply:
var ment som pseudokode, mulig det må lages et interface i tillegg